scripts/gas-refund-program/staking/2.0/utils.ts

```diff
@@ -13,3 +14,34 @@ export const fetchMigrationsTxHashesSet = async () => {
 
   return txHashes;
 };
+
+interface QueryFilterOptions {
+  batchSize: number;
+}
+export async function queryFilterBatched(
```

Comment:
ideally we should reimplement with sliceCalls

https://github.com/paraswap/paraswap-volume-tracker/blob/8068f73701b010bcb59c95d783b559a4a8d25791/src/lib/utils/helpers.ts#L21